File: Data Structures and Algorithms (Java & Python)/05 Hands On Activity 1/MovieTime.py
movies = []
snacks = []
i = 0
j = 0

while i < 3:
    movie = input("Enter movie " + str(i+1) + " of 3: ")
    movies.append(movie)
    i += 1
    
while j < 3:
    snack = input("Enter snack " + str(j+1) + " of 3: ")
    snacks.append(snack)
    j += 1
    
print("Movies to watch are: deque(" , movies , ")")
print("Snacks available are: deque(" , snacks , ")")
print("Press S each time you finish a snack.")

while True:
    finish = input()
    if finish == 's':
        snacks.pop(0)
        if not snacks:
            print("No more snacks.")
            break
    
    print("deque(" , snacks , ")")

File: Data Structures and Algorithms (Java & Python)/05 Hands On Activity 1/MovieTime.java
import java.util.Scanner;
import java.util.Queue;
import java.util.LinkedList;

public class MovieTime {
	public static void main(String[] args) {
		Queue<String> movies = new LinkedList<>();
		Queue<String> snacks = new LinkedList<>();
		Scanner scan = new Scanner(System.in);
		
		String movie;
		for(int i=0; i<3; i++) {
			System.out.print("Enter movie " + (i+1) + " of 3: ");
			movie = scan.nextLine();
			movies.add(movie);
		}
		String snack;
		for(int i=0; i<3; i++) {
			System.out.print("Enter snack " + (i+1) + " of 3: ");
			snack = scan.nextLine();
			snacks.add(snack);
		}

		System.out.println("Movies to watch are: deque(" + movies + ")");
		System.out.println("Snacks available are: deque(" + snacks + ")");

		System.out.println("Press S each time you finish a snack.");

		char finish;

		do {
			finish = scan.next().charAt(0);
			if (finish == 's') {
				snacks.remove();

				if (snacks.isEmpty()) {
					System.out.println("No more snacks.");
					break;
				}
			}
			System.out.println("deque(" + snacks + ")");
		} while (finish == 's');
	}
}